### DISCUSSION TITLE
Weblate 群员清理/销户问题

### DISCUSSION THREAD

**Discussion by TUsama** [CFPAOrg/Minecraft-Mod-Language-Package#905] [General]
讨论的主题有两个：
1. 如此做的必要性。
2. 如果真的需要如此做，那么具体的方案是如何的。

**Comment by 1478599553** [comment]
没有必要，除非wl账号特别占空间（当然不可能吧）

**Reply by WuzgXY-GitHub** [reply]
群员清理可能未来有需要

**Comment by TartaricAcid** [comment]
其实我还是挺喜欢 QQ 频道的设计，层次清晰功能全面
但是腾讯还是不公开正式发布，只能等待，qq 群本身还是太臃肿了。
如果未来 qq 频道公测了，而且普及开一段时间了，我倾向于直接转移到频道

**Reply by n0099** [reply]
快进到discord

**Reply by AnzhiZhang** [reply]
目前看起来遥遥无期，Discord/开黑啦是个不错的选择

对于多终端兼容性也更好，也可以直接公开加入
